src/FailureDetectorConfig.md

```diff
@@ -85,3 +88,59 @@ config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size",
 config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis", "100");
 [...]
 ```
+
+### Ping Failure Detector
+
+This failure detector is based on `InetAddress.isReachable()`.
+When the JVM process has enough permissions to create RAW sockets, the implementation will choose to rely on ICMP Echo requests, whereas, if not enough permissions, it will fallback on attempting a TCP Echo on port 7. In the latter, both a successful connection or an explicit rejection, will be treated as `Host is Reachable`.
+
+For the Ping Failure Detector to rely **only** on ICMP Echo requests, there are some criteria that need to be met.
+#### Requirements
+- Supported OS, as of Java 1.8 *nix environment support this
+- The Java executable must have the `cap_net_raw` capability. To do so, run `sudo setcap cap_net_raw=+ep path_to_java_exec`
+- When running with custom capabilities, the dynamic linker on Linux will reject loading libs from untrusted paths.
+To overcome, the `.../jre/lib/amd64/jli/` path needs to be added in the `ld.conf`
+`echo ".../jre/lib/amd64/jli/" >> /etc/ld.so.conf.d/java.conf && sudo ldconfig`
```

Comment:
Clarify those three dots. I suggest using this instead formulate it as:
To do so, execute:
echo "<JDK_HOME>/jre/lib/amd64/jli/" >> /etc/ld.so.conf.d/java.conf && sudo ldconfig 
where <JDK_HOME> is an absolute path to your JDK installation

Reply:
Or if JDK is not necessary, use <JRE_HOME>.